src/App.js

```diff
@@ -9,7 +10,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    <Article
+      title={article.title}
+      date={article.date}
+      text={article.text}
+    />
```

Comment:
```suggestion
    <Article { ...article } />
```
You could pass the whole object as a variant